public/css/styles.css

```diff
@@ -0,0 +1,49 @@
+.season{
+  text-align: center;
+}
+table{
+  border: 3px black;
+  border-collapse: collapse;
+  margin: 0 auto;
+  width: 80%;
+  
+}
+th{
+  background-color: #6495ee;
+  border: 3px solid black;
+  color: white;
+  margin: 0 auto;
+  padding: 5px;
+  text-align: center;
+ 
+}
+.num{
+  border: 3px solid black;
+  margin: 0 auto;
+  padding: 2px;
+  text-align: center;
+  width: 5%;
+
+}
+.title{
+  border: 3px solid black;
+  margin: 0 auto;
```

Comment:
You've got repetitive styles here that definitely could have been pulled out on their own